README.md

```diff
@@ -17,9 +17,12 @@ or
 ### Running manually or for journey tests
 
 <pre>
-sm --start CTC_TRADERS_DEPARTURE_P5_ACCEPTANCE -r OR
-sm --start CTC_TRADERS_DEPARTURE_P5_ACCEPTANCE CTC_TRADERS_API -r // with API running
+sm --start CTC_TRADERS_DEPARTURE_P5_ACCEPTANCE -r
```

Comment:
Obviously been a while since we updated this! That profile no longer exists as we did some work to consolidate our test suites into one profile. This should now be CTC_TRADERS_P5_ACCEPTANCE